src/Analysis/Problem/Bukkit/PluginDependenciesProblem.php

```diff
@@ -102,14 +83,13 @@ public static function getPatterns(): array
      */
     public function setMatches(array $matches, mixed $patternKey): void
     {
-        $this->pluginPath = $matches[1];
-        $this->pluginName = $matches[2];
-        $this->dependencyPluginNames = preg_split("/, ?/", $matches[3]);
+        $this->pluginName = $this->extractPluginName($matches[1]);
+        $this->dependencyPluginNames = preg_split("/, ?/", $matches[2]);
 
         foreach ($this->dependencyPluginNames as $name) {
             $this->addSolution((new PluginInstallSolution())->setPluginName($name));
         }
-        $this->addSolution((new FileDeleteSolution())->setRelativePath($this->getPluginPath()));
+        $this->addSolution((new PluginRemoveSolution())->setPluginName($this->getPluginName()));
```

Comment:
I think we should prefer providing the exact path of the plugin that should be deleted over only showing the plugin name.

Reply:
I added `BukkitPluginFileProblem` that stores the plugins file path and passes it to the `PluginRemoveFileSolution`.